### PR TITLE
Splunk doc

### PR DESCRIPTION
**Pull request by ahma**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] User guide and development docs updated (if needed)
- [x] Related Helm chart(s) updated (if needed)

### CODE REVIEW

File: docs/quickstarts/splunk.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+weight: 300
+---
+
+{{< contents >}}
+
+<p align="center"><img src="../../img/splunk.png" width="340"></p>
+
+This guide describes how to collect application and container logs in Kubernetes using the Logging operator, and how to send them to Splunk.
+
+The following figure gives you an overview about how the system works. The Logging operator collects the logs from the application, selects which logs to forward to the output, and sends the selected log messages to the output (in this case, to Splunk). For more details about the Logging operator, see the [Logging operator overview]({{< relref "docs/one-eye/logging-operator/_index.md">}}).
```
Comment (fekete-robert) [inline docs/quickstarts/splunk.md:13]: The following figure gives you ... : There is no figure here

File: docs/quickstarts/splunk.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+## Deploy Splunk
+
+First, deploy Splunk Standalone in your Kubernetes cluster. The following procedure is based on the [Splunk on Kubernetes quickstart](https://www.splunk.com/en_us/blog/it/an-insider-s-guide-to-splunk-on-containers-and-kubernetes.html)
```
Comment (fekete-robert) [inline docs/quickstarts/splunk.md:17]:
```suggestion
First, deploy Splunk Standalone in your Kubernetes cluster. The following procedure is based on the [Splunk on Kubernetes quickstart](https://www.splunk.com/en_us/blog/it/an-insider-s-guide-to-splunk-on-containers-and-kubernetes.html).
```

File: docs/quickstarts/splunk.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+## Deploy the Logging operator and a demo Application
+
+Install the Logging operator and a demo application to provide sample log messages.
+
+### Deploy the Logging operator with Kubernetes manifests
+
+To deploy the Logging operator using Kubernetes manifests, complete these steps. If you want to install the Logging operator using Helm, see [Deploy the Logging operator with Helm](#deploy-the-logging-operator-with-helm).
+
+1. Install the Logging operator. For details, see [How to install Logging-operator from manifests]({{< relref "docs/one-eye/logging-operator/deploy/_index.md#deploy-the-logging-operator-from-kubernetes-manifests" >}}
```
Comment (fekete-robert) [inline docs/quickstarts/splunk.md:57]:
```suggestion
1. Install the Logging operator. For details, see [How to install Logging-operator from manifests]({{< relref "docs/one-eye/logging-operator/deploy/_index.md#deploy-the-logging-operator-from-kubernetes-manifests" >}}.
```

File: docs/quickstarts/splunk.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+1. Create the `logging` resource.\
```
Comment (fekete-robert) [inline docs/quickstarts/splunk.md:58]:
```suggestion
1. Create the `logging` resource.
```

File: docs/quickstarts/splunk.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+    spec:
+      fluentd: {}
+      fluentbit: {}
+      controlNamespace: logging
+    EOF
+    ```
+
+     > Note: You can use the `ClusterOutput` and `ClusterFlow` resources only in the `controlNamespace`.
+
+1. Get Splunk HEC Token
```
Comment (fekete-robert) [inline docs/quickstarts/splunk.md:75]:
```suggestion
1. Get a Splunk HEC Token.
```

File: docs/quickstarts/splunk.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+
+     ```bash
+     HEC_TOKEN=$(kubectl get secret -n logging  splunk-single-standalone-secrets -o jsonpath='{.data.hec_token}' | base64 --decode)
+     ```
+
+1. Create Splunk output secret
```
Comment (fekete-robert) [inline docs/quickstarts/splunk.md:81]:
```suggestion
1. Create a Splunk output secret from the token.
```

File: docs/quickstarts/splunk.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+     ```bash
+     kubectl  create secret generic splunk-token -n logging --from-literal "SplunkHecToken=${HEC_TOKEN}"
+     ```
+
+
+1. Create an Splunk `output` definition.
```
Comment (fekete-robert) [inline docs/quickstarts/splunk.md:87]:
```suggestion
1. Define a Splunk `output`.
```

File: docs/quickstarts/splunk.md [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+
+To validate that the deployment was successful, complete the following steps.
+
+1. Use the following command to retrieve the password of the `admin` user:
+
+    ```bash
+    kubectl -n logging get secret splunk-single-standalone-secrets -o jsonpath='{.data.password}' | base64 --decode
+    ```
+
+1. Enable port forwarding to the Kibana Dashboard Service.
```
Comment (fekete-robert) [inline docs/quickstarts/splunk.md:173]:
```suggestion
1. Enable port forwarding to the Splunk Dashboard Service.
```

**Review by fekete-robert — CHANGES_REQUESTED** [review]

File: docs/quickstarts/splunk.md (hunk 1 above)
Comment (ahma) [reply docs/quickstarts/splunk.md]: Thanks!

**Review by asdwsda — APPROVED** [review]

**Review by fekete-robert — APPROVED** [review]

**Review by tarokkk — APPROVED** [review]

**Pull request merged**